contract/test/test-contract.js

```diff
@@ -20,15 +20,15 @@ test('zoe - mint payments', async (t) => {
   const bundle = await bundleSource(contractPath);
 
   // install the contract
-  const installation = await E(zoe).install(bundle);
+  const installation = E(zoe).install(bundle);
 
   const { creatorFacet, instance } = await E(zoe).startInstance(installation);
```

Comment:
What are the best practices for users to type their facets so that code hinting works?

Reply:
Issue here: https://github.com/Agoric/agoric-sdk/issues/3155